library/java/org/chromium/net/urlconnection/CronetBufferedOutputStream.java

```diff
@@ -0,0 +1,173 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
```

Comment:
Please remove licensing headers - that's not the way this is done with Envoy Mobile